### ISSUE TITLE
Ledger now use BIP44 for accounts address derivation

### ISSUE THREAD

**Issue by BrendanChou** [gnosis/MultiSigWallet#199]
https://github.com/MyCryptoHQ/MyCrypto/issues/2070

**Comment by GregTheGreek** [comment]
Any news on this?

**Comment by denisgranha** [comment]
You can use the ledger with the new derivation path trough metamask

**Comment by BrendanChou** [comment]
Ledger+Metamask is super buggy (borderline unusable)

**Comment by denisgranha** [comment]
Yeah it might be buggy, but should work. Is not a trivial change in our side, we have to support legacy path derivation and new ones, it might also involve updating the ledger library, and you know how this ecosystem is, inmature, it will break things we don't have time right now to solve because we are focused on the new Gnosis Safe.

I keep it open, as we would like to allow multiple derivation paths in the future.

**Comment by thegostep** [comment]
Ledger and metamask is no longer working

**Comment by denisgranha** [comment]
Hi @thegostep it’s actually a chrome thing, for newer ledger versions use brave browser...
